spec/requests/api/authentications_spec.rb

```diff
@@ -115,4 +115,27 @@
       expect(response).to have_http_status(:forbidden)
     end
   end
+
+  describe 'OPTIONS /api/authentications' do
+    it 'returns expected and additional attributes' do
+      api_basic_authorize
+
+      run_options(authentications_url)
+
+      attributes = (Authentication.attribute_names - Authentication.virtual_attribute_names).sort.as_json
+      attributes.delete('password')
+      reflections = (Authentication.reflections.keys | Authentication.virtual_reflections.keys.collect(&:to_s)).sort
+      subcollections = Array(Api::ApiConfig.collections[:authentications].subcollections).collect(&:to_s).sort
```

Comment:
the above four are now repeated (short of class name) in arbitration_rule_spec.rb, querying_spec.rb, and not authentications.  Can you create a helper that returns the four (attributes, virtual_attributes, relationships, subcollections) in a hash for a given class and use that helper ?  Thanks.